frontend/src/libs/api.ts

```diff
@@ -1,38 +1,10 @@
-import { POSTS_PAGE } from "../mocks/posts-page";
-import { POSTS_PAGE_PINNED } from "../mocks/posts-page-pinned";
-import { PostsId, User, UserMe } from "../types/common-types";
-
 export default async function api(endpoint: string, options?: RequestInit) {
+  let API_URL = `https://circleforms.net/api${endpoint}`;
   if (process.env.NODE_ENV === "development") {
-    const mocks = await import("../mocks");
-    await sleep(350);
-
-    if (endpoint.includes("/posts/page/pinned")) {
-      return POSTS_PAGE_PINNED;
-    }
-
-    if (endpoint.includes("/posts/page")) {
-      return POSTS_PAGE;
-    }
-
-    if (endpoint.includes("/posts/")) {
-      return mocks.form as PostsId;
-    }
-
-    if (endpoint.includes("/minimal")) {
-      return mocks.userMinimal as User;
-    }
-
-    if (endpoint === "/me/posts") {
-      return mocks.mePosts as PostsId[];
-    }
-
-    if (endpoint === "/me") {
-      return mocks.me as UserMe;
-    }
+    API_URL = `http://localhost/api${endpoint}`;
```

Comment:
this can be moved in an environment variable, this will allow us to change the endpoint without doing a if statement in the code

Documentation
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
Good call